08_algorithms.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Algorithms
+permalink: /algorithms/
+---
+
+SeqAn contains many efficient implementations of core bioinformatics algorithms. This starts with the standard dynamic programming based alignment algorithms with all its subtypes.
+<div style="flex-direction: row;">
+  <figure class="floating-figure">
+    <img src="/assets/images/overlay/algorithms_stellar.png">
+    <figcaption class="fig-caption">A depiction of the extension phase of the Stellar algorithm.</figcaption>
+  </figure>
+  <figure class="floating-figure">
+    <img src="/assets/images/overlay/algorithms_fiona.png">
+    <figcaption class="fig-caption">A depiction of the error correcting algorithm in the Fiona algorithm.</figcaption>
+  </figure>
```

Comment:
I'm not sure with floating them, since it does not look so good.

I wasn't aware of the figure element, looks good :)

I think we want to have one figure per row if we are on smartphone, but all of them on the same row otherwise.


![grafik](https://user-images.githubusercontent.com/103778/109853710-7eb8bd00-7c56-11eb-836e-09850b6e4d48.png)